docs/start/quickstart.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Quick Start
+position: 2
+---
+
+Infra is rolling out it's Software as a Service offering which makes it easier than ever to setup infrastructure access for your organization. This page will show you everything you need to do to get started.
+
+## _the tl;dr version_
```

Comment:
well, i didn't even notice the toc. I think its important for a user to see that this really is all that is needed, because they might see the text and think its hard.

Reply:
updated to point to the toc and say that really is all you need to do.